src/Kunstmaan/ApiBundle/Controller/NodesController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the KunstmaanBundlesCMS package.
+ *
+ * (c) Kunstmaan <https://github.com/Kunstmaan/KunstmaanBundlesCMS/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Kunstmaan\ApiBundle\Controller;
+
+use Doctrine\ORM\EntityManager;
+use FOS\RestBundle\Controller\ControllerTrait;
+use Nelmio\ApiDocBundle\Annotation\ApiDoc;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Class NodesController
+ *
+ * @author Ruud Denivel <ruud.denivel@kunstmaan.be>
+ *
+ * @Route(service="kunstmaan_api.controller.nodes")
+ */
+class NodesController
+{
+    use ControllerTrait;
+
+    /** @var EntityManager */
```

Comment:
Use `EntityManagerInterface` instead of the concrete implementation

Reply:
done